modelci/cli/modelhub.py

```diff
@@ -242,3 +244,16 @@ def delete(model_id: str = typer.Argument(..., help='Model ID')):
     with requests.delete(f'{app_settings.api_v1_prefix}/model/{model_id}') as r:
         if r.status_code == HTTPStatus.NO_CONTENT:
             typer.echo(f"Model {model_id} deleted")
+
+
+@app.command('convert')
+def convert(
+        id: str = typer.Option(..., '-i', '--id', help='ID of model.'),
```

Comment:
It's better to add more argument, such as:
- the model format use want to convert this model into, you can the use `convert` function
- the saved path of the converted model 
- whether to register the generated model into model hub

Reply:
I add 2 new arguments for savepath and register. I will add each specified convert after I make a revise to its function in the next PR.